wls-gui-wahllokalsystem/src/composables/common/rules.ts

```diff
@@ -63,6 +71,14 @@ export function useRules() {
     }
   };
 
+  function _formatToHhMm(compareValue: string): string {
```

Comment:
vorschlag zur umbenennung, damit klar wird, warum wir nicht die `toHhMm` funktion aus dem formatter nutzen:

```suggestion
  function _formatTimeStringToHhMm(compareValue: string): string {
```